objectRecognitionSource/README.md

```diff
@@ -462,9 +464,7 @@ The options are as follows. Remember to prepend "NN" when using an option in a Q
     * Example: `"anchors": [0.57273, 0.677385, 1.87446, 2.06253, 3.33843, 5.47434, 7.88282, 3.52778, 9.77052, 9.16828]`
 *   outputDir: Optional. Config and Query. The directory in which the images (object boxes included) will be placed.
     Images will be saved as "&lt;year&gt;-&lt;month&gt;-&lt;day&gt;--&lt;hour&gt;-&lt;minute&gt;-&lt;second&gt;.jpg"
-    where each value will zero-filled if necessary, e.g. "2018-08-14--06-30-22.jpg". For non-Queries, no images will
-    be saved if not set. For Queries, either this must be set in the Query, or this must be set in the config and
-    fileName must be set in the Query for images to be saved.
+    where each value will zero-filled if necessary, e.g. "2018-08-14--06-30-22.jpg".
```

Comment:
Are the other things (queries vs. non-queries) no longer true?

I guess this is described in more detail below.  Maybe add a link here that instructs the user that there's more detail elsewhere.

I suspect that it's worth mentioning that duplicates may have the counter.  Then the "see ... for more details" or something like that.